ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly-4.16-upgrade-from-eus-4.14.yaml

```diff
@@ -89,10 +89,6 @@ tests:
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       FILTERS_ADDITIONAL: ~TELCO&
       OCP_ARCH: arm64
-      UPGRADE_CHECK_RUN_TAGS: '@arm64 and @upgrade-check and @4.14 and @aws-ipi and
```

Comment:
Is this pr for both 4.12-4.14 and 4.14-4.16 or 4.12-4.14 only? If the former, there should be profile changes in amd/arm/multi jobs except for tags removing in 4.14-4.16 arm jobs.

Reply:
Actually I only want to update/add jobs for 4.12-4.14, I remove those tags is just because I noticed them and it does not have too much work.
I will changed back the tags.